src/pages/patientView/clinicalInformation/getClinicalInformationData.ts

```diff
@@ -21,13 +21,13 @@ function transformClinicalInformationToStoreShape(patientId: string, studyId: st
         clinicalData: clinicalDataPatient
     };
 
-    const samples: Array<ClinicalDataBySampleId> = chain(clinicalDataSample)
-        .groupBy('id')
-        .map((v, k) => ({
-            clinicalData: v,
-            id: k + '',
-        }))
-        .value();
+    const samples = _.map(
```

Comment:
I actually find the initial chained code to be more readable.  It's nice to avoid nesting and its nice to have the type right next to the variable we are populating.